.travis.yml

```diff
@@ -10,6 +10,10 @@ before_script:
   - cp keys/saml.key.enc.example keys/saml.key.enc
   - bin/rake db:setup --trace
   - bin/rake assets:precompile
+after_script:
+  - npm install -g pa11y-crawl
+  - bundle exec rails s -d
+  - pa11y-crawl --standard Section508 --ci http://localhost:3000
```

Comment:
I actually had issues with the _recommended_ method for starting the server; to use the `--run "rails server command"`